ChatQnA/ui/svelte/.env

```diff
@@ -1 +1 @@
-DOC_BASE_URL = 'http://localhost:8000/v1/rag'
```

Comment:
Is localhost enough?

Reply:
The .sh script will run in the background, automatically fill in the URL into the .env file, and then run the frontend code. Currently, it's the most basic framework to ensure the overall process flows smoothly.